README.md

```diff
@@ -9,19 +9,26 @@ Turn Twitch chatter into Python events.
 ## Features
 
 - *Pure Python:* No extra dependencies. Just plain and simple Python.
-- *Small API:* With two classes and four methods, you can learn it over a coffee break.
+- *Small API:* With three classes and seven methods, you can learn it over a coffee break.
 - *Event Based:* Makes writing apps easy and straightforward.
 - *Context Manager:* Further simplifies working with observers.
 
 ## Usage
 
 ### 1. Installation
 
-To install twitch-observer clone from the [repo](https://github.com/JoshuaSkelly/twitch-observer) and use:
+To install twitch-observer clone from the [repo](https://github.com/JoshuaSkelly/twitch-observer) and run:
 
 ```
 $ pip install .
 ```
+
+You can also use the makefile by typing:
+
+```
+$ make
```

Comment:
I don't think we need to mention the make file. Folks who are used to using those will see it and use it.